go-dlv.el

```diff
@@ -131,6 +134,27 @@ and source-file directory for your debugger."
   (setq paragraph-start comint-prompt-regexp)
   (run-hooks 'go-dlv-mode-hook))
 
+;;;###autoload
+(defun dlv-current-func ()
+  "Debug the current program or test stopping at the beginning of the current function."
+  (interactive)
+  (setq test-name nil)
+  (ignore-errors (setq test-name (go-test-get-current-test)))
```

Comment:
What kind of errors do you expect here? It is better to ensure the error won't happen with appropriate checks.